### PR TITLE
Runner: New App - initial commit

### PR DESCRIPTION
**Pull request by zamastyle**
**Note: The min phantom version 4.9.0 is intended. Kindly refer below comments for more info.**

Please ensure your pull request (PR) adheres to the following guidelines:

- Please refer to our contributing documentation for any questions on submitting a pull request, link: [Contribution Guide](https://github.com/splunk-soar-connectors/.github/blob/main/.github/CONTRIBUTING.md)

#### Please check if your PR fulfills the following requirements:
- [x] Testing of all the changes has been performed (for bug fixes / features)
- [x] The readme.html has been reviewed and added / updated if needed (for bug fixes / features)
- [x] Use the following format for the PR description: `<App Name>: <PR Type> - <PR Description>`
- [x] Provide release notes as part of the PR submission which describe high level points about the changes for the upcoming GA release.
- [x] Verify all checks are passing.
- [x] Do NOT use the `next` branch of the forked repo. Create separate feature branch for raising the PR.
- [x] Do NOT submit updates to dependencies unless it fixes an issue.
#### Please check the type of change your PR introduces:
- [x] New App
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation
- [ ] Other (please describe): 

## Security Considerations (REQUIRED)
- [ ] If you are exposing any endpoints using a [REST handler](https://docs.splunk.com/Documentation/SOAR/current/DevelopApps/RESTHandlers), 
  please document them in the `readme.html`.
- [ ] If this is a new connector or you are adding new actions
    - [ ] Please document in the `readme.html` all methods (eg, OAuth) used to authenticate 
      with the service that the connector is integrating with.
    - [ ] If any actions are unable to run on SOAR Cloud, please document this in the `readme.html`.
- [ ] Are you introducing any new cryptography modules? If yes, please elaborate their purpose:
- [ ] Are you are accessing the file system? If yes, please verify that you are only accessing paths returned through 
the [Vault](https://docs.splunk.com/Documentation/SOARonprem/5.2.1/DevelopApps/AppDevAPIRef#Vault) API.
- [ ] Are you are marking code to be ignored by Semgrep with [`nosemgrep`](https://semgrep.dev/docs/ignoring-files-folders-code/#ignoring-code-through-nosemgrep)?
If yes, please provide justification in an additional comment next to the ignored code.

  

## Release Notes (REQUIRED)
- Provide release notes as part of the PR submission which describe high level points about the changes for the upcoming GA release.

-- initial commit for the Runner utility app

## What is the current behavior? (OPTIONAL)
- Describe the current behavior that you are modifying.

## What is the new behavior? (OPTIONAL)
- Describe the behavior or changes that are being added by this PR.

## Other information (OPTIONAL)
- Any other information that is important to this PR such as screenshots of how the component looks before and after the change.

## Pay close attention to (OPTIONAL)
- Any specific code change or test case points which must be addressed/reviewed at the time of GA release.

## Screenshots (if relevant)

---
Thanks for contributing!

### CODE REVIEW

**Comment by zamastyle** [comment]
Ready for review. All errors have been reviewed
"Min Number Log Statements" - using custom logging function (__print) to pass messages to save_progress and debug_print
"Licenses" - unknown what this requirements.txt file should look like
"Min Platform Version" - apps have existed for years and tested on many versions of phantom. 4.9.0 is intentional
"Repo Name Has Expected App Id" - unknown what the format of this json file should be
"App Actions" - action name " clear scheduled playbooks" should remain as is
"Verbose Periods" - I'm pretty sure there are periods at the end of every verbose description

**Comment by dgopani-crest** [comment]
Hi @zamastyle, we have checked your PR and added in our queue. We will soon start working on this and release the app

**Comment by zamastyle** [comment]
@mishalp-crest 
The logo updates are fine. thanks.
The min phantom version was intended to be 4.9.0 though. please change that back to 4.9.0.

**Comment by mishalp-crest** [comment]
> @mishalp-crest The logo updates are fine. thanks. The min phantom version was intended to be 4.9.0 though. please change that back to 4.9.0.

@zamastyle 
Phantom version 5.2 is more stable and as per the standard process, we have to add the min phantom version 5.2.0.

**Comment by zamastyle** [comment]
> 

Then this will be a non-standard change. Please change it back to 4.9.0 as intended.

**Comment by mishalp-crest** [comment]
> > 
> 
> Then this will be a non-standard change. Please change it back to 4.9.0 as intended.

@zamastyle reverted!

**Comment by zamastyle** [comment]
@mishalp-crest 
Perfect

**Comment by ighelani-crest** [comment]
@mishalp-crest 
License/Copyright strings are missing in all the files. Apache 2.0 License should be added to the required files. Could you please check this?

File: readme.html [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<!-- File: readme.html
+  Copyright (c) Mhike, 2022.
+  Licensed under Apache 2.0 (https://www.apache.org/licenses/LICENSE-2.0.txt)
```
Comment (achandya-crest) [inline readme.html:3]: Add complete License string

**Comment by mishalp-crest** [comment]
> @mishalp-crest License/Copyright strings are missing in all the files. Apache 2.0 License should be added to the required files. Could you please check this?

@ishans-crest done

File: readme.html (hunk 1 above)
Comment (mishalp-crest) [reply readme.html:3]: @achandya-crest done!

**Review by ighelani-crest — APPROVED** [review]

**Review by achandya-crest — APPROVED** [review]

**Review by alexa-phantom — APPROVED** [review]

**Pull request merged**